docs/index.rst

```diff
@@ -294,3 +361,68 @@ A single page dialog appears that allows you to change any of the parameters of
 Once you have updated your notification click *Save* to action the changes.
 
 If you wish to delete your notification this may be done by clicking the *Delete* button at the base of the dialog.
+
+Statistics Rate Example
+-----------------------
+
+A simple example of the use of statistics for a notification is to
+monitor the flow of data within a pipeline. We can use the statistics
+that the north service updates for the data processed by the north
+service to determine if the service is functioning and sending data out
+of the Fledge instance. In this case the name of the statistic we want
+to monitor is the same as the name of the service.
```

Comment:
Can we use `Readings sent` for total sent as well? or only service name are allowed.